tests/plugins/os/unix/bsd/osx/test__os.py

```diff
@@ -25,7 +25,7 @@ def test_unix_bsd_osx_os(target_osx_users, fs_osx):
 
     assert dissect_user.name == "_dissect"
     assert dissect_user.passwd == "*"
-    assert dissect_user.home == "/var/empty"
+    assert str(dissect_user.home) == "/var/empty"
```

Comment:
It should not be needed either way as the flow path `__eq__` can deal with directly comparing against strings.

Reply:
The point is to bypass the overriden `__eq__` method 

By comparing to  the string representation when the test is running under windows, I am ensuring  that the home directory is stored in posix_format; otherwise, it would fail because the string representation would contain "\".

Do you want me to add a comment?

Alternatively, i can do a type assertion for  `posix_path`

Reply:
Added comment